gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -1006,3 +1007,38 @@ def get_sample_qc_field_def_json_path(version: str = CURRENT_VERSION) -> str:
     :return: Path to sample QC field definitions JSON.
     """
     return f"{get_sample_qc_root(version)}/additional_resources/gnomad.exomes.v{version}.sample_qc_field_definitions.json"
+
+
```

Comment:
I would prefer that all of these be located under the v4 resources path. `gs://gnomad/v4.0/sample_qc/additional_resources`